src/chainstate/burn/db/sortdb.rs

```diff
@@ -726,7 +780,8 @@ const SORTITION_DB_SCHEMA_4: &'static [&'static str] = &[
 // update this to add new indexes
 const LAST_SORTITION_DB_INDEX: &'static str = "index_burn_header_hash_pox_valid";
```

Comment:
You'll need to change this to `index_peg_out_fulfill_burn_header_hash` so the code knows to add the new indexes.

Reply:
Done ✅